operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -40,29 +40,75 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
   * Exact split with Justine to be determined
 
-Note: Few areas not listed above, such as the [User's Guide](https://docs.mattermost.com/guides/user.html), will remain a collaborative effort between Product Managers and Technical Writers. Please assume @jason.blais as the directly responsible individual.
+Note: Some areas not listed above, such as the [User's Guide](https://docs.mattermost.com/guides/user.html), will remain a collaborative effort between Product Managers and Technical Writers. Please assume @jason.blais as the directly responsible individual.
 
 ## Processes
 
 The processes around documentation, such as SLAs for issues and requesting documentation assistance are detailed below.
 
-### **Contributing to Documentation**
+### Contributing to Documentation
 
 The basic outline for getting started with contributions is provided in the [README](https://github.com/mattermost/docs/blob/master/README.md) of the docs repo. If you have write access to the repo, you can create a branch off master and work on that.
 
-Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/) - it applies to various types of contributions including documentation.
+Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review) - it applies to various types of contributions including documentation.
 
-### DocUp/Documentation Issue Process
+### Submitting Documentation With Your PR
 
-Requests for documentation can be made within [community.mattermost.com](https://community.mattermost.com), using the DocUp plugin embedded in the post menu.
+We want to empower everyone to contribute to our documentation, and be comfortable submitting documentation for contributions. As such, we don't expect every contribution to adhere to our style guide. 
+
+Here are some guidelines around this:
+
+* When submitting a PR, please include updated documentation if applicable. 
+* The documentation update can be in the form of a bullet list or an outline.
+* Label the PR as **Docs Needed** and tag @justinegeffen or @amyblais.
+* The documentation you've submitted will be reviewed.
+* At times, the editors may make and commit stylistic changes \(such as punctuation\) but any content changes will be added as a suggestion for the submitter's consideration. 
+* Once the PR is approved, it will be merged, and the documentation will be updated. 
+
+You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review). 
+
+Note: This process does not apply to the [API Documentation requirements](https://github.com/mattermost/mattermost-api-reference). 
+
+### Adding Reviewers to PRs
```

Comment:
@jasonblais, I added dev reviewers based on the list here: https://developers.mattermost.com/internal/rd-teams/. Open to suggestions regarding the areas I've linked them to and/or ones that aren't listed but should be.

Reply:
That's a good start, we can have Dean Whillier as the dev reviewer for `End User Guide - Other: `